### ISSUE TITLE
Nan Loss During the training

### ISSUE THREAD

**Issue by yilin-wu98** [personalrobotics/pytorch_retinanet#9]
Hi,

I am training the model with the food data with 16 different food item labels as described in your [paper](https://personalrobotics.cs.washington.edu/publications/feng2019generalization.pdf). However, during the training, in the first epoch, around batch_idx = 1900, the loss becomes Nan. Do you have any suggestions on why this is happening?
